.github/workflows/build-test.yml

```diff
@@ -50,13 +56,15 @@ jobs:
       - name: "flake8"
         shell: "bash -l {0}"
```

Comment:
The `shell: "bash -l {0}"` lines can now be dropped (likewise elsewhere), it was only needed for setup-miniconda.

Reply:
Thanks!